test/acceptance/registry.js

```diff
@@ -167,7 +167,7 @@ describe('registry', function(){
       it('should return the component with custom headers', function() {
         expect(result[0].response.version).to.equal('1.0.0');
         expect(result[0].response.name).to.equal('hello-world-custom-headers');
-        expect(result[0].response.headers).to.be.deep.equal({'cache-control': 'public max-age=3600', 'test-header': 'Test-Value'});
+        expect(result[0].headers).to.be.deep.equal({'cache-control': 'public max-age=3600', 'test-header': 'Test-Value'});
```

Comment:
Yeah agree. I think the purposed change here is to move headers one level down on the POST so that instead of having
```json
{   
  components: [{
    status: 200,
    response: {
      ...
      headers: { ... }
    }
  }]
}
```
we have 
```json
{   
  components: [{
    status: 200,
    response: {
      ...
    },
    headers: { ... }
  }]
}
```